openssl.yaml

```diff
@@ -2,10 +2,15 @@
 package:
   name: openssl
   version: 3.4.0
-  epoch: 4
+  epoch: 5
   description: "the OpenSSL cryptography suite"
   copyright:
     - license: Apache-2.0
+  dependencies:
+    # For apk upgrades
+    replaces:
+      - openssl-config<3.4.0
```

Comment:
yes; once this lands openssl-config.apk must no longer be used; which itself is EOL and no longer updated package....... at some point there were no references to openssl-config, and yes I am adding this because they could have crept back in.

Reply:
To track: https://github.com/chainguard-dev/internal-dev/issues/7799